bundles/org.openhab.binding.meater/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,162 @@
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<context>password</context>
+				<description>The password used to login to MEATER Cloud account</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="10" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds</description>
+				<default>30</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="meaterprobe">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="meaterapi"/>
+		</supported-bridge-type-refs>
+
+		<label>MEATER Probe</label>
+		<description>This thing represents a MEATER Probe</description>
+
+		<channels>
+			<channel id="internalTemperature" typeId="internalTemperature"/>
+			<channel id="ambientTemperature" typeId="ambientTemperature"/>
+			<channel id="cookId" typeId="cookId"/>
+			<channel id="cookName" typeId="cookName"/>
+			<channel id="cookState" typeId="cookState"/>
+			<channel id="cookTargetTemperature" typeId="cookTargetTemperature"/>
+			<channel id="cookPeakTemperature" typeId="cookPeakTemperature"/>
+			<channel id="cookElapsedTime" typeId="cookElapsedTime"/>
+			<channel id="cookRemainingTime" typeId="cookRemainingTime"/>
+			<channel id="cookEstimatedEndTime" typeId="cookEstimatedEndTime"/>
+			<channel id="status" typeId="status"/>
+			<channel id="lastConnection" typeId="lastConnection"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Apption Labs</property>
+			<property name="model">MEATER</property>
+		</properties>
+
+		<representation-property>deviceId</representation-property>
+
+		<config-description>
+			<parameter name="deviceId" type="text" required="true">
+				<label>Device Id</label>
+				<description>Unique id for your MEATER Probe</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="status">
+		<item-type>String</item-type>
+		<label>Current Status</label>
+		<description>Information on current status</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="lastConnection">
+		<item-type>DateTime</item-type>
+		<label>Last Probe Connection</label>
+		<description>Date and time of last probe connection</description>
+		<category>Time</category>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM"/>
+	</channel-type>
+
+	<channel-type id="cookElapsedTime">
+		<item-type>Number:Time</item-type>
+		<label>Cook Elapsed Time</label>
+		<description>Time since the start of cook in seconds. Default: 0</description>
+		<category>Time</category>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="cookRemainingTime">
+		<item-type>Number:Time</item-type>
+		<label>Cook Remaining Time</label>
+		<description>Remaining time in seconds. When unknown/calculating default is used. Default: UnDefType.UNDEF</description>
```

Comment:
I still don't think "default" is the correct word to use here. Since **UNDEF** is pretty common for unknown/undefined/irrelevant values, I would propose to leave this out of the description and shorten to simply:
```suggestion
		<description>Remaining time in seconds.</description>
```

If you insist on mentioning this, perhaps something like:
```suggestion
		<description>Remaining time in seconds or UNDEF when unknown.</description>
```

Reply:
Fixed!